SU-HSDO/suhumsci
# Archaeology and France-Stanford Subthemes

# READY FOR REVIEW

# Summary
- Adds a subtheme for Archaeology and France-Stanford
- Moves above site's specific code out of the suhumsci base theme
- Cleans up lingering site-specific settings and config in base theme

# Needed By (Date)
- End of sprint (10/25)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Sync
3. Go to `admin/appearance` and enable related subtheme; make sure site-specific styles appear

# Affected Projects or Products
- Archaeology
- France-Stanford
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSD8-461
- https://stanfordits.atlassian.net/browse/HSD8-462

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
